arm-ci.sh

```diff
@@ -12,6 +12,6 @@ elif [ "$__buildArch" == "armel" ]; then
     __dockerImage=" hqueue/dotnetcore:ubuntu1404_cross_prereqs_v4-tizen_rootfs"
     __dockerEnvironmentVariables+=" -e ROOTFS_DIR=/crossrootfs/armel.tizen.build"
 fi
-__dockerCmd="sudo docker run ${__dockerEnvironmentVariables} --privileged -i --rm -v $__currentWorkingDirectory:/opt/code -w /opt/code $__dockerImage"
+__dockerCmd="sudo docker run ${__dockerEnvironmentVariables} --privileged --cap-add=ALL -v /dev:/dev -v /lib/modules:/lib/modules -i --rm -v $__currentWorkingDirectory:/opt/code -w /opt/code $__dockerImage"
```

Comment:
what is `/dev` needed for?

Reply:
Removed, this was due to difference between my local machine and CI.